[jdbc] clean item with force leading to NPE

`jdbc tables clean test_item force` 
Leads to an NPE shown down below

```
2022-12-11 12:46:04.677 [DEBUG] [persistence.jdbc.internal.JdbcMapper] - JDBC::ifTableExists
2022-12-11 12:46:04.678 [DEBUG] [istence.jdbc.internal.db.JdbcBaseDAO] - JDBC::doIfTableExists sql=SHOW TABLES LIKE 'item0156'
2022-12-11 12:46:04.682 [DEBUG] [persistence.jdbc.internal.JdbcMapper] - JDBC::deleteItemsEntry
2022-12-11 12:46:04.683 [ERROR] [b.core.io.console.ConsoleInterpreter] - An error occurred while executing the console command.
java.lang.NullPointerException: replacement
	at java.util.regex.Matcher.replaceFirst(Matcher.java:1403) ~[?:?]
	at java.lang.String.replaceFirst(String.java:2081) ~[?:?]
	at org.openhab.persistence.jdbc.internal.utils.StringUtilsExt.replaceArrayMerge(StringUtilsExt.java:58) ~[?:?]
	at org.openhab.persistence.jdbc.internal.db.JdbcBaseDAO.doDeleteItemsEntry(JdbcBaseDAO.java:344) ~[?:?]
	at org.openhab.persistence.jdbc.internal.JdbcMapper.deleteItemsEntry(JdbcMapper.java:148) ~[?:?]
	at org.openhab.persistence.jdbc.internal.JdbcPersistenceService.cleanupItem(JdbcPersistenceService.java:407) ~[?:?]
	at org.openhab.persistence.jdbc.internal.JdbcPersistenceService.cleanupItem(JdbcPersistenceService.java:372) ~[?:?]
	at org.openhab.persistence.jdbc.internal.console.JdbcCommandExtension.cleanupItem(JdbcCommandExtension.java:165) ~[?:?]
	at org.openhab.persistence.jdbc.internal.console.JdbcCommandExtension.execute(JdbcCommandExtension.java:105) ~[?:?]
```

It looks like 
`public static final String replaceArrayMerge(String str, String[] separate, String[] separators)`
is called where seperate and seperators do not match in array size. The util class does not have any check on same array length.

This is waith jdbc / mariadb and 3.4.0 M5

The command `jdbc tables list` gives:
```
item0156          5  test_item Item missing
```

--- a/bundles/org.openhab.persistence.jdbc/src/main/java/org/openhab/persistence/jdbc/internal/JdbcPersistenceService.java
+++ b/bundles/org.openhab.persistence.jdbc/src/main/java/org/openhab/persistence/jdbc/internal/JdbcPersistenceService.java
@@ -525,6 +525,7 @@ public class JdbcPersistenceService extends JdbcMapper implements ModifiablePers
                 if (!conf.getTableUseRealCaseSensitiveItemNames()) {
                     ItemsVO itemsVo = new ItemsVO();
                     itemsVo.setItemName(entry.getItemName());
+                    itemsVo.setItemsManageTable(conf.getItemsManageTable());
                     deleteItemsEntry(itemsVo);
                 }
                 itemNameToTableNameMap.remove(entry.getItemName());

